components/JoinUnimindSection/JoinUnimindSection.tsx

```diff
@@ -9,6 +10,8 @@ import { Button } from "../Button/Button";
 
 const JoinUnimindSection = () => {
   const theme = useTheme();
+  const { t } = useTranslation("join-unimind");
+  const { t: tCommon } = useTranslation("common");
```

Comment:
you can pass 2 namespaces into useTrasnslation like `const { t } = useTranslation(["join-unimind", "common"]);`

and then use it like `t("join-unimind:title")` :)

Reply:
Changed